packages/compass-smoke-tests/tsconfig.json

```diff
@@ -3,7 +3,11 @@
   "compilerOptions": {
     "noEmit": true,
     "types": ["node"],
-    "lib": ["ES2021"]
+    "lib": ["ES2021"],
+    "module": "NodeNext",
```

Comment:
Needed so that typescript doesn't compile the dynamic import to a require and then complain that we're require()-ing an ES module.